sync/linux/controlplane.sh

```diff
@@ -154,6 +154,8 @@ cp $HOME/.kube/config /var/sync/shared/config
 rm -f /var/sync/shared/kubejoin.ps1
 
 cat << EOF > /var/sync/shared/kubejoin.ps1
+cp C:\Users\vagrant\crictl.exe "C:\Program Files\containerd"
```

Comment:
This one deserve a single patch for quick merge.
Also, I would recomment: `mv C:\Users\vagrant\crictl.exe "C:\Program Files\containerd"` instead of cp.

Reply:
@aravindhp added you as author, just signed as well as I was submitting:
https://github.com/kubernetes-sigs/sig-windows-dev-tools/pull/145

Reply:
Thanks for doing that